dashboard/app/views/levels/_single_multi.html.haml

```diff
@@ -1,10 +1,10 @@
-.multi{id: "multi_#{index}"}
+- data = level.properties
+- app = 'multi'
 
-  - data = level.properties
-  - app = 'multi'
+- # setting this so that multi_t works
+- @level = level
 
-  - # setting this so that multi_t works
-  - @level = level
+.multi{id: "level_#{index}"}
```

Comment:
I built this out, but levelGroup.js needs a way to grab results from each of the objects, so using a simple index from 0..numLevels-1 seems easier...